code/modules/power/cell.dm

```diff
@@ -280,6 +280,7 @@
 	maxcharge = 50000
 	rating = 12
 	ratingdesc = FALSE
+	self_recharge = 1

```

Comment:
@GunHog As I said, EMPs don't actually do anything to infinite cells.
The SMES issue wouldn't be solved by preventing EMPs as the infinite cell will be treated as having 50,000 charge, not inifnite, which is still wrong.